SDL/Segment.h

```diff
@@ -965,27 +965,27 @@ namespace SDL {
           eta_number = -1;
 
         if (category_number == 0 && eta_number == 0)
-          occupancy = 572;
+          occupancy = 927;
         else if (category_number == 0 && eta_number == 1)
-          occupancy = 300;
+          occupancy = 349;
         else if (category_number == 0 && eta_number == 2)
-          occupancy = 183;
+          occupancy = 253;
         else if (category_number == 0 && eta_number == 3)
-          occupancy = 62;
+          occupancy = 61;
         else if (category_number == 1 && eta_number == 0)
-          occupancy = 191;
+          occupancy = 1011;
         else if (category_number == 1 && eta_number == 1)
-          occupancy = 128;
+          occupancy = 653;
```

Comment:
some updates are going up quite a bit, like here. On the other hand some others like in T5 https://github.com/SegmentLinking/TrackLooper/pull/397/files#diff-d9d3d7c519ab64f27b6e501281a0eb282be964d2802ddca40387348a68bb6cb0R3182 are going down

Cat1 is layers 4,5,6. It looks like something like layer 6 or even 5 already opens up a lot.
On one hand, an additional category may be useful.
On the other, it would be good to see some plots of the segment angles (is it phi change): there was a discussion to truncate this so that it doesn't hit pi/2 or even larger.

Reply:
> Cat1 is layers 4,5,6. It looks like something like layer 6 or even 5 already opens up a lot.

given this is a segment; the possible layers are 4 and 5.